pyproject.toml

```diff
@@ -17,7 +17,7 @@ requests = "^2"
 click = "^8"
 pydantic = "<2"
 websocat = "^1.12.0"
-golem-node = "0.12.3"
+golem-node = "0.13.1"
```

Comment:
whoah, do we want to upgrade yagna to `0.13.1` already?

Reply:
Reverted